packages/server/modules/core/rest/upload.ts

```diff
@@ -1,34 +1,43 @@
-'use strict'
-const zlib = require('zlib')
-const { corsMiddleware } = require('@/modules/core/configs/cors')
-const Busboy = require('busboy')
-
-const { validatePermissionsWriteStream } = require('./authUtils')
-const {
+import zlib from 'zlib'
+import { corsMiddleware } from '@/modules/core/configs/cors'
+import Busboy from 'busboy'
+import { validatePermissionsWriteStream } from '@/modules/core/rest/authUtils'
+import {
   getFeatureFlags,
   maximumObjectUploadFileSizeMb
-} = require('@/modules/shared/helpers/envHelper')
-const {
+} from '@/modules/shared/helpers/envHelper'
+import {
   createObjectsBatched,
   createObjectsBatchedAndNoClosures
-} = require('@/modules/core/services/objects')
-const { ObjectHandlingError } = require('@/modules/core/errors/object')
-const { estimateStringMegabyteSize } = require('@/modules/core/utils/chunking')
-const { toMegabytesWith1DecimalPlace } = require('@/modules/core/utils/formatting')
+} from '@/modules/core/services/objects'
```

Comment:
Do make sure to test this, sometimes imports don't act as expected when converting from require() to import(), e.g. `const XXX = require('asd')` is not necessarily always translated to a default import.